resources/js/xcoin.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * @link https://www.coinsence.org/
+ * @copyright Copyright (c) 2019 Coinsence
+ * @license https://www.humhub.com/licences
+ *
+ * @author Ghaith Daly <daly.ghaith@gmail.com>
+ */
+humhub.module('xcoin', function (module, require, $) {
+
+    var client = require('client');
+
+    var loader = '<div class="loader humhub-ui-loader" style="padding:0">' +
+        '<div class="sk-spinner sk-spinner-three-bounce">' +
+        '<div class="sk-bounce1"></div>' +
+        '<div class="sk-bounce2"></div>' +
+        '<div class="sk-bounce3"></div>' +
+        '</div></div>';
+
+    $("body").on('click', '#ether-enable-btn' ,function () {
```

Comment:
As discussed yesterday we should consider improve event handling in future releases